client/package.json

```diff
@@ -26,7 +25,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "postbuild": "rm -rf ../server/frontend_bootstrap/build && mkdir ../server/frontend_bootstrap/build && mv ./build/* ../server/frontend_bootstrap/build/",
+    "postbuild": "rm -rf ../server/frontend_bootstrap/build && mkdir ../server/frontend_bootstrap/build && mv ./build/* ../server/frontend_bootstrap/build/ && mv ../server/frontend_bootstrap/build/static/* ../server/frontend_bootstrap/build/",
```

Comment:
I understand this is REALLY weird.

This is due to an issue in django (???) whereby having a path like `static/static` confuses django w.r.t. whether the path is relative or absolute (e.g. it thinks that `static/static/some/path` refers to a relative path, and `static/some/path` refers to an absolute path) . Miraculously, if we just... remove the "static" layer from the build file WITHOUT updating the internal references, django resolves the directory and everything just works.

This is obviously SUPER BRITTLE and dependent on what I believe naively to be a bug in Django. I'd love to figure out something better than this.